src/OWML.ModHelper.Menus/ModPopupInput.cs

```diff
@@ -15,12 +15,12 @@ public abstract class ModPopupInput<T> : ModInput<T>
 		protected readonly TwoButtonToggleElement ToggleElement;
 		protected ModCommandListener CommandListener;
 
-		private readonly List<SingleAxisCommand> _openCommands = new()
-		{
-			InputLibrary.menuConfirm,
-			InputLibrary.enter,
-			InputLibrary.enter2
-		};
+		//private readonly List<SingleAxisCommand> _openCommands = new()
+		//{
+		//	InputLibrary.menuConfirm,
+		//	InputLibrary.enter,
+		//	InputLibrary.enter2
+		//};
```

Comment:
Hm... maybe add some kind of attribute/comment to everything we comment out so we don't miss anything?

Reply:
we can search for "//" ;)